tensornetwork/tn_keras/dense.py

```diff
@@ -122,3 +125,31 @@ def f(x: tf.Tensor, a_var: tf.Tensor, b_var: tf.Tensor, use_bias: bool,
 
   def compute_output_shape(self, input_shape: List[int]) -> Tuple[int, int]:
     return (input_shape[0], self.output_dim)
+
+  def get_config(self) -> dict:
+    """Returns the config of the layer.
+
+    The same layer can be reinstantiated later
+    (without its trained weights) from this configuration.
+
+    Returns:
+      Python dictionary containing the configuration of the layer.
+    """
+    # Get base config
+    config = super(DenseDecomp, self).get_config()
+
+    # Include the DenseDecomp-specific arguments
+    decomp_args = ['output_dim', 'decomp_size', 'use_bias']
+    for arg in decomp_args:
+      config[arg] = getattr(self, arg)
+
+    # Serialize the activation
+    config['activation'] = activations.serialize(getattr(self, 'activation'))
+
+    # Serialize the initializers
+    decomp_initializers = ['kernel_initializer', 'bias_initializer']
+    for initializer_arg in decomp_initializers:
+      config[initializer_arg] = initializers.serialize(
+          getattr(self, initializer_arg))
+
+    return config
```

Comment:
Because you're passing a **kwargs into super(), you need to be sure to get the superclass config dict as well. The idiomatic Keras for that is:
    config = {...}
    base_config = super(CLS, self).get_config()
    return dict(list(base_config.items()) + list(config.items()))

Reply:
Agreed - at present I get the superclass config on line 139: `config = super(DenseDecomp, self).get_config()` and then add to it. Is there a particular reason to keep `base_config` and `config` separate until the last line? If yes, I'll switch to that approach.

Reply:
The only reason would be to match the expression in other Keras layers - I missed what your code was doing :) Up to you though.

Reply:
Sounds good! Updated to the idiomatic Keras for posterity :)